spark-neoforge/src/main/java/me/lucko/spark/neoforge/NeoForgeWorldInfoProvider.java

```diff
@@ -34,32 +34,16 @@
 import net.minecraft.world.level.ChunkPos;
 import net.minecraft.world.level.GameRules;
 import net.minecraft.world.level.entity.EntityLookup;
-import net.minecraft.world.level.entity.EntitySection;
-import net.minecraft.world.level.entity.EntitySectionStorage;
+import net.minecraft.world.level.entity.LevelEntityGetter;
 import net.minecraft.world.level.entity.PersistentEntitySectionManager;
 import net.minecraft.world.level.entity.TransientEntitySectionManager;
 
-import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
-import java.util.stream.Stream;
+import net.neoforged.fml.ModList;
 
 public abstract class NeoForgeWorldInfoProvider implements WorldInfoProvider {
 
-    protected List<ForgeChunkInfo> getChunksFromCache(EntitySectionStorage<Entity> cache) {
-        LongSet loadedChunks = cache.getAllChunksWithExistingSections();
-        List<ForgeChunkInfo> list = new ArrayList<>(loadedChunks.size());
-
-        for (LongIterator iterator = loadedChunks.iterator(); iterator.hasNext(); ) {
-            long chunkPos = iterator.nextLong();
-            Stream<EntitySection<Entity>> sections = cache.getExistingSectionsInChunk(chunkPos);
-
-            list.add(new ForgeChunkInfo(chunkPos, sections));
-        }
-
-        return list;
-    }
-
```

Comment:
Please could you also refactor the forge module so we use the same approach there (appreciate moonrise will not be supported tho)